ksql-engine/src/test/java/io/confluent/ksql/ddl/commands/DropSourceCommandTest.java

```diff
@@ -111,18 +97,19 @@ public void shouldFailOnDropIncompatibleSource() {
     dropSourceCommand.run(metaStore);
   }
 
-  private void givenDropSourceCommand(final boolean ifExists, final boolean deleteTopic) {
+  private void givenDropSourceCommand(final boolean ifExists) {
     dropSourceCommand = new DropSourceCommand(
-        new DropStream(QualifiedName.of(STREAM_NAME), ifExists, deleteTopic),
-        StructuredDataSource.DataSourceType.KSTREAM,
-        deleteTopic);
+        new DropStream(QualifiedName.of(STREAM_NAME), ifExists,
```

Comment:
I take it the value of `isDelete` in the `DropStream` constructor is not irrelevant for this test?  So consider hardcoding it and dropping `WITHOUT_DELETE_TOPIC`...